src/inverseRobot.js

```diff
@@ -1,13 +1,33 @@
 'use strict';
 
-/*
- * @param {object} robot
- *
- * @return {object}
-*/
+const robert = {
+  Robert: 'name',
+  123: 'chipVer',
+  113: 'chipVerPro',
+};
+
+inverseRobot(robert);
 
 function inverseRobot(robot) {
-  // write code here
+  const inversedRobot = {};
+
+  if (robot.hasOwnProperty('name') && robot.name !== 'name') {
+    inversedRobot[robot.name] = 'name';
+  }
+
+  for (const objectKey in robot) {
+    const objectValue = robot[objectKey];
```

Comment:
```suggestion
  for (const key in robot) {
    const value = robot[key];
```
No need to add "object" before name